csharp-basics/exercises/Polymorphism/DragRace/Bmw.cs

```diff
@@ -31,12 +31,12 @@ public string ShowCurrentSpeed()
 
         public void UseNitrousOxideEngine()
         {
-            currentSpeed+=30;
+            currentSpeed += 30;
         }
-
-        public void StartEngine()
+        
```

Comment:
Tukšus space nevajag atstāt

Reply:
Apskatoties vēlreiz to failu es neatradu liekus space
![image](https://user-images.githubusercontent.com/61376013/134910728-1033c2f2-84ca-41d6-b538-fad283c9c50e.png)

Reply:
Ir extensions, kurš parāda, kur ir lieki space, bet tos far pameklē arī ar meklētēju ievadot 2 spacus un tad iekrāsos.